dramatiq/middleware/age_limit.py

```diff
@@ -46,5 +46,4 @@ def before_process_message(self, broker, message):
 
         if current_millis() - message.message_timestamp >= max_age:
             self.logger.warning("Message %r has exceeded its age limit.", message.message_id)
-            message.fail()
```

Comment:
I think the right change here is to retain the `message.fail()` line _and_ `raise SkipMessage`.